packages/main/src/styles/global.css

```diff
@@ -188,6 +188,11 @@ h6 {
     background-color: var(--bg-2);
     left: 0px;
   } */
+/* background color change for modal */
+[data-test-id="layout-wrapper"]{
+  background: var(--primary-color) !important;
```

Comment:
important?

Reply:
Yes, the styles were added inline and this is the only way to override